.github/workflows/macos.yml

```diff
@@ -0,0 +1,39 @@
+name: macOS latest
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    env:
+      PACKAGE: ignition-sensors4
```

Comment:
We could consider using a combination of `GITHUB_REPOSITORY` and `CMakeLists` to deduce this, like the [Linux CI does](https://github.com/ignition-tooling/action-ignition-ci/blob/8b919064d2f2f3f5a828d5e5e71288cd793c909e/entrypoint.sh#L40-L42).

This way, this action could be moved to a common place and reused for other Ignition libraries. This should help maintenance, although this looks pretty straightforward already, and may not change much. We may for example need to add extra code checks or dependencies, and then need to touch all libraries to do so. I also thought we may not change https://github.com/ignition-tooling/action-ignition-ci much, but it turns out we often do little tweaks to it.

Just something to consider. We don't need to block this in case you think it's not worth it.

Reply:
I think that is a great idea, but I'd prefer to leave that task for another PR, though we could try to do it before back porting this

Reply:
SGTM :+1: